src/js/components/SkipLinkAnchor.js

```diff
@@ -0,0 +1,23 @@
+// (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
+
+var React = require('react');
+var IntlMixin = require('../mixins/GrommetIntlMixin');
+
+var SkipLinkAnchor = React.createClass({
+
+  propTypes: {
+    label: React.PropTypes.string.isRequired
+  },
+
+  mixins: [IntlMixin],
+
+  render: function () {
+    var id = 'skip-link-' + this.props.label.toLowerCase().replace(/ /g, '_');
+    return (
+      <div tabIndex="0" id={id} data-skip-label={this.getGrommetIntlMessage(this.props.label)} className="skip-link-anchor" />
```

Comment:
Given that this seems to be a link anchor, is there a reason we aren't using an <a> instead of a <div>?